packages/rspeedy/plugin-react/api-extractor.json

```
@@ -4,6 +4,7 @@
 {
   "extends": "../../../api-extractor.json",
   "bundledPackages": [
-    "@lynx-js/react-transform",
+    "@lynx-js/swc-plugin-compact",
+    "@lynx-js/swc-plugin-react-lynx",
```

This doesn't seem to take effect.
```suggestion
    "@lynx-js/swc-plugin-reactlynx-compat",
    "@lynx-js/swc-plugin-reactlynx",
```